src/Agent.Plugins/TFCliManager.cs

```diff
@@ -33,11 +33,11 @@ public override TfsVCFeatures Features
 
         protected override string Switch => "/";
 
-        public string FilePath => Path.Combine(ExecutionContext.Variables.GetValueOrDefault("Agent.ServerOMDirectory")?.Value, "tf.exe");
+        public string FilePath => Path.Combine(ExecutionContext.Variables.GetValueOrDefault("Agent.HomeDirectory")?.Value, "externals", "tf", "tf.exe");
```

Comment:
- do we ever add tf to the path?
- do we expect customers are looking for tf in a well known location? will that break or is it still in the om directory too?